### PR TITLE
Added a new lobbyscreen

### PR DESCRIPTION
**Pull request by elisarek**
## About the PR
Added a new lobby screen

## Technical details

-Added survivalharmony.webp and its associated YML file into /Resources/Textures/LobbyScreens.
-Added survivalharmony.webp to lobbyscreens.yml
-Added survivalharmony.webp to attributions.yml in /Resources/Textures/LobbyScreens under the CC-BY-NC-SA-4.0 license.
## Media

Ss14_Harmony

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl:
- add: Added "Survival Harmony" lobby art by Logan Avery

### CODE REVIEW

File: Resources/Textures/LobbyScreens/survivalharmony.webp.webp
Comment (FluffMe) [inline Resources/Textures/LobbyScreens/survivalharmony.webp.webp:1]: File name is incorrect.
Please rename to `survivalharmony.webp`
(currently is `survivalharmony.webp.webp`)

File: Resources/Textures/LobbyScreens/attributions.yml
Comment (FluffMe) [inline Resources/Textures/LobbyScreens/attributions.yml:74]: Should be `https://github.com/ss14-harmony/space-station-14` unless it's from upstream

**Review by FluffMe — CHANGES_REQUESTED** [review]
Almost there! Couple of changes pending.

Great art, thank you for doing this.

File: Resources/Textures/LobbyScreens/survivalharmony.webp.webp
Comment (elisarek) [reply Resources/Textures/LobbyScreens/survivalharmony.webp.webp:1]: 🤦no clue how that happened, many thanks.

**Review by FluffMe — APPROVED** [review]
Looks good. ~~Will merge after test pass~~ I am not the art code owner as I have no taste. So we wait for Keld.

Please draw more stuff!

**Review by KeldWolf — APPROVED** [review]
Approved

**Pull request merged**
